cwms-data-api/src/test/java/cwms/cda/data/dao/location/kind/EmbankmentDaoIT.java

```diff
@@ -78,33 +83,39 @@ public static void setup() throws Exception {
             } catch (IOException e) {
                 throw new RuntimeException(e);
             }
-        });
+        },
+        CwmsDataApiSetupCallback.getWebUser());
     }
 
     @AfterAll
-    public static void tearDown() throws Exception {
+    public void tearDown() throws Exception {
```

Comment:
Pretty sure AfterAll requires static

Reply:
See previous comment. Arguable I should change it back, but frankly things are working now so I'd rather not risk breaking them again.